common/src/main/java/com/wynntils/models/statuseffects/StatusEffectModel.java

```diff
@@ -34,10 +34,18 @@ public final class StatusEffectModel extends Model {
      *
      * <p>Originally taken from: <a href="https://github.com/Wynntils/Wynntils/pull/615">Legacy</a>
      */
-    private static final Pattern STATUS_EFFECT_PATTERN = Pattern.compile(
-            "(?<prefix>.+?)(?<modifier>§7 ?([%\\-+\\.\\/\\d]+s?)?) *(?<name>[a-zA-Z\\s]+?) (?<timer>§[84a]\\((.+?)\\)).*");
+    /*
+     * current regex:
+     * <prefix>         captures any characters before the §7 colour-indicator.
+     * <modifier>       optionally captures effects' modifiers' values (+100, -20, 5)
+     * <modifierSuffix> optionally captures the suffixes of the modifiers (/5s, /3s, %)
+     * <name>           captures the effects name (Strength, and even +Lightweight)
+     * <timer>          captures the effects duration ( (xx:xx) or (**:**) ).
+     *
```

Comment:
👍 

We should really have descriptions like this on more regexps. (They are powerful, but tend to be write-only...)